test/system/notifications_bell_test.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+
+class NotificationsBellTest < ApplicationSystemTestCase
+  test 'Unread and All tabs filter their target notifications' do
+    visit_with_auth '/notifications', 'komagata'
+    click_link '全て既読にする'
+
+    Notification.create(message: 'お知らせのテスト通知',
+                        kind: 'announced',
+                        link: '/announcements/1',
+                        user: users(:komagata),
+                        sender: users(:machida))
```

Comment:
```suggestion
    Notification.create(
      message: 'お知らせのテスト通知',
      kind: 'announced',
      link: '/announcements/1',
      user: users(:komagata),
      sender: users(:machida)
    )
```

こちらのフォーマットの方が横幅も短くて良いかもです〜。

Reply:
ありがとうございます。こちらの方が読みやすいと思いました。
他Issueでテストを書く際も意識しようと思います。
このフォーマットで修正いたしました。